pkgs/by-name/xc/xcbuild/package.nix

```diff
@@ -42,6 +42,9 @@
 #'' true;
 
 let
+  # xcbuild is included in the SDK. Avoid an infinite recursion by using a bootstrap stdenv.
+  stdenv' = if stdenv.hostPlatform.isDarwin then darwin.bootstrapStdenv else stdenv;
```

Comment:
Same as `atf`.

Frankly, these should probably go in the overlays in `pkgs/stdenv/darwin`, though, as it is knowledge based on the specific set‐up of that `stdenv`, which can change over time.